javascript-local-storage/javascript-local-storage-quiz-notes.md

```diff
@@ -0,0 +1,41 @@
+# javascript-local-storage-quiz-notes
+
+## Quiz Questions
+
+After completing this exercise, you should be able to discuss or answer the following questions:
+
+- How do you store data in `localStorage`?
+
+- How do you retrieve data from `localStorage`?
+
+- What data type can `localStorage` save in the browser?
+
+- When does the `'beforeunload'` event fire on the `window` object?
```

Comment:
Don't forget the quiz questions!

Reply:
Sorry, I keep forgetting these! I'll make sure to do better at checking everything before I put in a PR.